unstructured/ingest/cli/cmds/azure.py

```diff
@@ -7,43 +7,57 @@
 from unstructured.ingest.cli.interfaces import (
     CliConfig,
 )
+from unstructured.ingest.connector.azure import (
+    AzureAccessConfig,
+    AzureWriteTextConfig,
+    SimpleAzureBlobStorageConfig,
+)
 
 CMD_NAME = "azure"
 
 
 @dataclass
-class AzureCliConfig(CliConfig):
-    account_id: t.Optional[str] = None
-    account_name: t.Optional[str] = None
-    connection_string: t.Optional[str] = None
-
+class AzureCliConfig(AzureAccessConfig, CliConfig):
     @staticmethod
     def get_cli_options() -> t.List[click.Option]:
         options = [
             click.Option(
                 ["--account-key"],
                 default=None,
-                help="Azure Blob Storage or DataLake account key (not required if "
-                "`azure_account_name` is public).",
+                help="The storage account key. This is used for shared key "
+                "authentication. If any of account key, sas token or "
+                "client_id is specified, anonymous access will be used.",
             ),
             click.Option(
                 ["--account-name"],
                 default=None,
-                help="Azure Blob Storage or DataLake account name.",
+                help="The storage account name. This is used to authenticate "
+                "requests signed with an account key and to construct the "
+                "storage endpoint. It is required unless a connection string "
+                "is given, or if a custom domain is used with anonymous authentication.",
             ),
             click.Option(
                 ["--connection-string"],
                 default=None,
-                help="Azure Blob Storage or DataLake connection string.",
+                help="If specified, this will override all other parameters besides "
+                "request session. See http://azure.microsoft.com/en-us/"
+                "documentation/articles/storage-configure-connection-string/ "
+                "for the connection string format.",
+            ),
+            click.Option(
+                ["--sas_token"],
+                default=None,
+                help="A shared access signature token to use to authenticate requests "
+                "instead of the account key. If account key and sas token are both "
+                "specified, account key will be used to sign. If any of account key, "
+                "sas token or client_id are specified, anonymous access will be used.",
```

Comment:
Is this a typo, it says anonymous access will be used when credentials are provided

Reply:
I actually copied these texts from the source library but I think they might have a typo themselves, will fix: [adlfs/spec.py#L127](https://github.com/fsspec/adlfs/blob/main/adlfs/spec.py#L127)